src/python/CRABClient/JobType/PrivateMC.py

```diff
@@ -31,7 +31,7 @@ def run(self, *args, **kwargs):
             self.logger.debug("LHESource found in the CMSSW configuration.")
             configArguments['generator'] = getattr(self.config.JobType, 'generator', 'lhe')
 
-            major, minor, fix = os.environ['CMSSW_VERSION'].split('_')[1:4]
+            major, minor, _ = os.environ['CMSSW_VERSION'].split('_')[1:4]
```

Comment:
OK. I found the answer by myself, sorry for noise. `_` is indeed Python conding convention for throwaway variable. I learnt one more thing.